cmd/chainbridge/main.go

```diff
@@ -188,6 +189,10 @@ func run(ctx *cli.Context) error {
 		c.AddChain(newChain)
 	}
 
+	go func() {
+		health.Start(c)
```

Comment:
`Start()` doesn't seem to block, go routine is unnecessary

Reply:
The http server is blocking within `Start()`